pkg/component/logging/vali/vali.go

```diff
@@ -133,7 +133,6 @@ type Values struct {
 	CuratorImage          string
 	TelegrafImage         string
 	KubeRBACProxyImage    string
```

Comment:
This block needs to be formatted after this change.

Reply:
Yes, `make generate` takes care of that. Thank you.